src/components/Search.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import "../styles/Search.css";
+import propTypes from "prop-types";
+import getImages from "../requests/getImages";
+
+const Search = ({ setSearchResults }) => {
+  const [value, setValue] = useState();
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    setSearchResults(await getImages(value));
+  };
+
+  return (
+    <div className="Search">
+      <form className="search-form" onSubmit={handleSubmit}>
+        <input
+          className="search-input"
+          type="text"
+          onChange={(e) => setValue(e.target.value)}
+        />{" "}
+        <br /> <br />
```

Comment:
`<br>` really shouldnt be used here, could use the margin property for either of the adjacent elements to achieve the same thing. Also what is the {" "} for?

Reply:
Removed br here and used CSS instead, also removed {" "}.